### ISSUE TITLE
Latest Version of `remark-toc` Uses ESM Modules

### ISSUE THREAD

**Issue by RehanSaeed** [gridsome/gridsome#1539]
## Summary

The latest version of `remark-toc` uses ESM modules. I'm unable to upgrade to it, I think Gridsome needs to update itself.

```
// From CommonJS
var remarkToc = require('remark-toc')

// To ESM
import remarkToc from 'remark-toc'
```

**Comment by songololo** [comment]
Heads-up that this is also rippling through the remark ecosystem as other packages also update to ESM.

**Comment by NickSto** [comment]
Yeah it's a huge pain. Kludge I've been using: https://www.npmjs.com/package/jiti
